components/Footer.tsx

```diff
@@ -1,27 +1,25 @@
 /** @jsx jsx */
 import * as React from "react";
 import { jsx } from "@emotion/core";
-import { Box, Flex, TextLink } from "@mojotech/mojo-ui";
-import Logo from "./Logo";
+import { Wrap, Flex, TextLink } from "@mojotech/mojo-ui";
 
 const Footer: React.FC = () => (
-  <Box background="#1A1723" as="footer">
+  <Wrap as="footer">
     <Flex
       flexDirection={["column", "row"]}
-      m="0 auto"
       py={4}
-      px={3}
-      maxw={[0, 3]}
+      justifyContent="space-around"
```

Comment:
i think `space-between` is the way to go so that there isnt space outside the items, we want them extending to the edge of the wrap no?